Makefile

```diff
@@ -3,6 +3,7 @@ GID = $(shell id -g)
 DOCKER_IMAGE = pim-docs
 DOCKER_RUN = docker run -it --rm -u $(UID):$(GID) -v $(PWD):/home/akeneo/pim-docs/data
 DOCKER_RSYNC = $(DOCKER_RUN) -v /etc/passwd:/etc/passwd:ro -v $${SSH_AUTH_SOCK}:/ssh-auth.sock:ro -e SSH_AUTH_SOCK=/ssh-auth.sock $(DOCKER_IMAGE) rsync -e "ssh -q -p $${DEPLOY_PORT} -o StrictHostKeyChecking=no" -qarz --delete
+DOCKER_BUILD = $(DOCKER_RUN) -v $(PWD)/_build:/home/akeneo/ce
```

Comment:
Why creating a new alias when it seems to not be used anywhere in the Makefile?

Reply:
mix of intentions... I started from Laure's branch 
nice catch!